.circleci/config.yml

```diff
@@ -760,15 +752,27 @@ workflows:
           context:
             - GCP2
           mad-dog-type: test
-      - hold-test-mad-dog-e2e-full:
-          type: approval
-      - test-mad-dog-e2e:
-          context:
-            - GCP2
-          name: test-mad-dog-e2e-full
-          mad-dog-type: test-nightly
-          requires:
-            - hold-test-mad-dog-e2e-full
+
+      # Commenting this out since we almost never run this job, and it ruins our nice green check marks.
```

Comment:
can we just del?

Reply:
figured i'd leave in for now and let infra decide if they want to del or not

Reply:
otherwise they'd have to go find the appropriate commit and revert, blah blah

Reply:
I would prefer if no build jobs happened manually on developer machines. Ideally, all of our deployment processes would be included in our CI pipeline.

Given that CIrcleCI is going to make green checkmarks better "soon", I would hope the commented out code would be temporary.

[Related thread](https://audius-internal.slack.com/archives/C03CGM2EML4/p1654886447006159).

Reply:
agreed, up to you / infra what you think is best, but i think this is a valuable stopgap